docs/help/account/billing.md

```diff
@@ -0,0 +1,49 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. We use [Stripe](https://stripe.com/) to handle all payments, so we never store your bank details.
+
+:::info
+Payment is made by credit card only. We are considering offering alternative payment methods in the future.
+:::
+
+## Invoices
+
+### Edit your billing information
+
+You can edit the following billing information so that it appears on your upcoming invoices:
+
+- Company name
+- Address
+- VAT Number
+
+:::info
+Updating your billing information will only apply to future invoices; it is not possible to edit the details of an already created invoice.
+:::
+
+### Invoices
+
+Every month, we send your invoice in PDF format to the email address linked to your account. You can specify an additionnal email address to which they will be sent, for your accounting department, for example.
+
+On this page, you will also find a history of your invoices, available in PDF.
+
+### Taxes
+
+We apply taxes in accordance with current regulations. Three scenarios are possible:
+
+- your billing address is located outside of Europe: we do not apply VAT.
+- your billing address is located in Europe (except France): we do not apply VAT.
```

Comment:
Nope : we do apply 20% VAT for European accounts not providing a VAT number. We do not apply VAT for European (except France) accounts providing a VAT number.

This is a crazy rule, I know 😂 (but not our choice, this is what our accountant told us to implement)